packages/flutter/lib/src/cupertino/text_selection_toolbar.dart

```diff
@@ -950,8 +988,10 @@ class _RenderCupertinoTextSelectionToolbarItems extends RenderBox with Container
     _backButton!.layout(constraints.loosen(), parentUsesSize: true);
     _nextButton!.layout(constraints.loosen(), parentUsesSize: true);
 
-    final double subsequentPageButtonsWidth =
-        _backButton!.size.width + _nextButton!.size.width;
+    // Require the children to have at least the height of the nav buttons.
+    final double minHeight = clampDouble(constraints.minHeight, _backButton!.size.height, _nextButton!.size.height);
```

Comment:
Do we still need `minHeight` now that we're vertically centering them?

Reply:
Good point there. I was able to remove this.